packages/material-ui-codemod/src/v5.0.0/mui-replace.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * @param {import('jscodeshift').FileInfo} file
+ */
+export default function transformer(file) {
+  return file.source
+    .replace(/@material-ui\/unstyled/gm, '@mui/core')
+    .replace(/@material-ui\/core/gm, '@mui/material')
+    .replace(/@material-ui\/lab/gm, '@mui/lab-material')
```

Comment:
What @mnajdova used so far:

```suggestion
    .replace(/@material-ui\/lab/gm, '@mui/lab')
```

Reply:
Are we still discussing the package names?

Reply:
Jun asked about this package specifically in https://github.com/mui-org/material-ui/pull/28049#issuecomment-908383758. I personally agree with Marija, the lab is about being able to release breaking changes forever. It's broader than Material Design.

Reply:
Let's keep it `@mui/lab` then